docs/conf.py

```diff
@@ -171,6 +171,9 @@
             margin-bottom: 0.4em;
             line-height: 2em;
         }
+        .nboutput:has(.prompt.empty):has(.output_area:empty) {
+            display: none;
+        }
```

Comment:
Is there a better fix? You should be able to run the workflow once my PR is merged- https://github.com/tardis-sn/tardis/pull/3075

I feel like this is more like a patch but doesn't really solve the root problem.

Reply:
Yep, I am searching for other options. For some reason, the logger widget duplicated on one notebook, so I wanted to confirm whether it is just a bug on my system or not.